src/App.vue

```diff
@@ -44,15 +47,14 @@ export default {
       const user = confirmLogin(userData, this.allUsers)
       if (user) {
         this.updateLogin()
-        this.currentUser = user.id
+        setUserId(user.id)
       }
     },
     createNewUser(userData) {
       submitNewUser(userData)
-      .then(data => this.currentUser = data)
-      this.updateLogin()
-    },
-  },
+      .then(data => setUserId(data.id))
+    }
+  }
```

Comment:
Ope, had to replace all the "this.currentUser"s haha. Worth it in the end though!